.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2.1
 executors:
   node:
     docker:
-      - image: cimg/node:16.13
+      - image: cimg/node:18.12
```

Comment:
note: I'm not using the latest one (which is 18.14 currently), because it's not available in appveyor yet, and I'd have to install it with `Update-NodeJsInstallation` which is fairly slow.